### PR TITLE
Bulk upload errors

### PR DESCRIPTION
**Pull request by proteusvacuum**
http://manage.dimagi.com/default.asp?217420#1092816

@orangejenny

### CODE REVIEW

File: corehq/apps/app_manager/views/translations.py [hunk 1]
```diff
@@ -42,7 +42,6 @@ def _html_message(header_text, messages):
             message += "<li>%s</li>" % prop
         return message
 
-    messages.error(request, _("Welcome to CommCare."))
```
Comment (czue) [inline corehq/apps/app_manager/views/translations.py]: lol

**Comment by czue** [comment]
build fail looks real (i wonder if it explicitly tests that these are removed)
```
======================================================================
FAIL: test_translation (corehq.apps.app_manager.tests.test_bulk_ui_translation.BulkUiTranslation)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/corehq/apps/app_manager/tests/test_bulk_ui_translation.py", line 66, in test_translation
    u'home.start.demo': u'Dong'}
AssertionError: {u'fra': {u'entity.sort.title': u'wubble', u'unknown_string': u'Dong', u'home.st [truncated]... != {u'fra': {u'entity.sort.title': u'wubble', u'home.start.demo': u'Dong', u'activi [truncated]...
  {u'en': {u'date.tomorrow': u'wobble',
           u'entity.sort.title': u'wabble',
-          u'home.start.demo': u'Ding',
+          u'home.start.demo': u'Ding'},
?                                     +
-          u'unknown_string': u'Ding'},
   u'fra': {u'activity.locationcapture.Longitude': u'wibble',
            u'entity.sort.title': u'wubble',
-           u'home.start.demo': u'Dong',
?                                      ^
+           u'home.start.demo': u'Dong'}}
?                                      ^^
-           u'unknown_string': u'Dong'}}
----------------------------------------------------------------------
```

**Comment by proteusvacuum** [comment]
:ballot_box_with_check:

File: corehq/apps/app_manager/views/translations.py (hunk 1 above)
Comment (benrudolph) [inline corehq/apps/app_manager/views/translations.py]: hahaha salty

**Pull request merged**